packages/wrangler/src/__tests__/pages/pages-deployment-tail.test.ts

```diff
@@ -80,10 +102,11 @@ describe("pages deployment tail", () => {
 
 			await api.closeHelper();
 			expect(api.requests.deletion.count).toStrictEqual(1);
+			await api.closeHelper();
```

Comment:
For some reason this didn't work in an `afterEach()`... I've copied this line to the end of every test that needs it, but longer-term we should find a better way to test websockets, since this approach has layers upon layers of mocks